lib/cartodb/api/version.rb

```diff
@@ -1,5 +1,5 @@
-module Cartodb
+module CartoDB
   module Api
-    VERSION = "0.1.0"
+    VERSION = '0.0.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.